kotlin-bom/build.gradle

```diff
@@ -0,0 +1,7 @@
+plugins {
+    id "io.micronaut.build.internal.bom"
+}
+
+micronautBom {
+    extraExcludedProjects.add(project(":examples:greeting").name)
```

Comment:
is this project used in the docs?

Reply:
Yes it is, but I don't think it should be published, nor in the BOM?

Reply:
yes, it should not be published and not in the bom. Maybe it would be worth renaming to `test-suite-kotlin` as we do in `security` and that way we don't need to do this. Not sure what you think

Reply:
It doesn't seem to be a test suite, but an example, so I think the current setup is fine.